chaos/net-rpc-client.go

```diff
@@ -75,10 +76,7 @@ func (rpcClient *ReconnectRPCClient) dialRPCClient() (*rpc.Client, error) {
 }
 
 // Call makes a RPC call to the remote endpoint using the default codec, namely encoding/gob.
-func (rpcClient *ReconnectRPCClient) Call(serviceMethod string, args interface {
```

Comment:
dsync does not know/worry about how server/client is authenticated.   It facilitates networked sync.mutex.

In Minio server use case, minio will do authentication based on JWT+timestamp in `lock-client` to `lock-server` where lock-client is interface compatible to `dsync.NetLocker`.

Let me add a token based Login/Logout method in chaos test for sake of understanding.